js/general.js

```diff
@@ -33,51 +33,56 @@ shinyjs.init = function() {
 
```

Comment:
In general we should go over this sometime, my understanding is near zero here

Reply:
Turbo-noted ^_^